src/vue-froala.js

```diff
@@ -271,7 +271,7 @@ export default (Vue, Options = {}) => {
     }    
   };
 
-  Vue.component('froala', froalaEditorFunctionality);
+  Vue.component('Froala', froalaEditorFunctionality);
```

Comment:
We could do that, though we should also keep the other one for compatibility reasons for a while.

Reply:
We don't break a compatibility.
> This PR not contains breaking changes, because if component registered in PascalCase, we can use both approaches - kebab-case and PascalCase.

Reference:
https://vuejs.org/v2/guide/components-registration.html#Name-Casing

![Documentation Screenshot](https://image.prntscr.com/image/LMt3HF7WQbSYbBBPWWuyTA.png)